src/main/java/com/epam/tc/hw9/service/YandexSpellerService.java

```diff
@@ -0,0 +1,27 @@
+package com.epam.tc.hw9.service;
+
+import com.epam.tc.hw9.dto.YandexSpellerDto;
+import com.epam.tc.hw9.utils.Parameters;
+import com.google.gson.Gson;
+
+
+public class YandexSpellerService extends CommonService {
+
+    public YandexSpellerDto[] getCorrectWord(String text) {
```

Comment:
Better to use names from service documentation. Service has only two methods checkText and checkTexts. 
Also current name _'getCorrectWord'_ lead to confusion.